src/amoc_api_scenario_handler.erl

```diff
@@ -81,8 +81,7 @@ resource_exists(Req, State = #state{resource = Resource}) ->
 -spec to_json(cowboy_req:req(), state()) -> 
     {jsx:json_text(), cowboy_req:req(), state()}.
 to_json(Req0, State = #state{resource = Resource}) ->
-    Status = amoc_controller:test_status(
-                     erlang:list_to_atom(Resource)),
+    Status = amoc_dist:test_status(erlang:list_to_atom(Resource)),
```

Comment:
Ad 1. It's safe here because when we want to do a `GET` request cowboy checks whether scenario exists on filesystem, so in this line we will get a name of a existing module (but maybe not loaded)
Ad 2. We use `binary_to_list/2` in `rest_init` function to check later in `resource_exists` whether our scenario is on disk. Without that we would need to convert our binary/atom to list, so in my opinion it's not a problem.